### ISSUE TITLE
ReHLDS is not a recreated "mod"

### ISSUE THREAD

**Issue by a1batross** [hlsources/hlsources.github.io#1]
Hi! Thanks for this list, actually a quite nice one. :)

I found that you added ReHLDS into recreated mods category which isn't true. They are re-implementing dedicated server. It could be added to GoldSrc Engine category, as it's completely compatible with it.

**Comment by Maxxiii** [comment]
Hello, thank you. Sorry for really long response - only after 1.5 years ;) Just today I accidentally noticed errors on nemstools archive website, and then noticed that related issue was already reported on github last september and I didn't get any notifications about it. Apparently, github doesn't send notifications for Owners of organizations (https://alexking.org/blog/2011/11/28/not-getting-github-notifications), which is really strange choice. For now I just turned on  related checkboxes inside "Watch" menu for main repositories, so I hope this won't happen again.
Speaking of ReHLDS, you're right, it's not really mod, so I should move it to GoldSrc category, thank you for suggestion. Will do it tomorrow probably, with some other additions I wanted to make, since several mods were added to the list on Xash3D FWGS documentation page.
